gnuradio-runtime/lib/qa_circular_file.cc

```diff
@@ -24,9 +24,8 @@
 #include "config.h"
 #endif
 
-#include "qa_circular_file.h"
 #include "circular_file.h"
-#include <cppunit/TestAssert.h>
+#include <boost/test/unit_test.hpp>
 #include <iostream>
```

Comment:
You know, while we're at it... Remove, if you'd like to make your maintainer extra happy

Reply:
Yeah this isn't on the latest version.